site/src/pages/_app.page.tsx

```diff
@@ -20,6 +20,7 @@ import {
   UserContextValue,
   UserState,
 } from "../context/user-context";
+import { defaultSeoConfig } from "../default-seo-config";
```

Comment:
It's helpful to keep things that are used together close to each other. Extracting a separate file makes sense if we plan to use `defaultSeoConfig` outside `_app.page.tsx`. I suggest defining this constant right in the file, but I may be missing something. What benefits of a separate file to you see?